Core/src/IfcGeometryConverter/SolidModelConverter.h

```diff
@@ -1015,7 +1026,7 @@ namespace OpenInfraPlatform
 
 					if (axisPlacement->Location)
 					{
-						axisLocation = placementConverter->convertIfcCartesianPoint(axisPlacement->Location );
+						axisLocation = placementConverter->convertIfcPoint(axisPlacement->Location);
 					}

```

Comment:
I assume this can be greatly simplified by using specialized functions in PlacementConverter.